e2e/e2e_test.go

```diff
@@ -40,7 +40,7 @@ func init() {
 	flag.BoolVar(&testRBD, "test-rbd", true, "test rbd csi driver")
 	flag.BoolVar(&helmTest, "helm-test", false, "tests running on deployment via helm")
 	flag.BoolVar(&upgradeTesting, "upgrade-testing", false, "perform upgrade testing")
-	flag.StringVar(&upgradeVersion, "upgrade-version", "v3.3.1", "target version for upgrade testing")
+	flag.StringVar(&upgradeVersion, "upgrade-version", "v3.5.1", "target version for upgrade testing")
```

Comment:
@humblec can you are fixing this part can you please check if you can take care of https://github.com/ceph/ceph-csi/issues/2818?

Reply:
sure, let me check that :+1: